src/main/cpp/io/util/Usb.cpp

```diff
@@ -65,6 +66,18 @@ void Usb::Open()
         throw IOException("(%04X:%04X) Claiming interface %d failed: %s", m_vid, m_pid, m_interface,
             libusb_error_name(ret));
     }
+
+    //needed this for LXIO, need to test is it causes issues for PIUIO
+    ret = libusb_set_interface_alt_setting(m_dev, m_interface, 0);
+
+    if (ret != LIBUSB_SUCCESS) {
+        libusb_close(m_dev);
+        libusb_exit(m_ctx);
+
+        throw IOException("(%04X:%04X) Setting interface alternate: %s", m_vid, m_pid, libusb_error_name(ret));
+    }
+
+    
```

Comment:
Wasn't sure how to handle that. I still need to test with a mk6 PIUIO, as it may not actually affect anything there.

Reply:
I thought you mentioned that this affects PIUIO negatively. If that's not the case, I don't see any issues with having this swapped like in your proposal. Maybe add an in-code comment just in case to point this out if this breaks for anything else in the future.

Reply:
Tested with piuiomk6, and still works without issue.